server/api/actions/actions.api.spec.js

```diff
@@ -109,22 +111,56 @@ describe('Actions API', function() {
     });
   });
 
-  /**
-   * Returns an object representing the expected properties of a Location, based on the specified Location.
-   * (Can be used, for example, to check if a returned API response matches a Location in the database.)
-   *
-   * @param {action} action - The action to build the expectation from.
-   * @param {...Object} changes - Additional expected changes compared to the specified Location (merged with Lodash's `extend`).
-   * @returns {Object} An expectations object.
-   **/
-  function getExpectedAction(action, ...changes) {
-    return _.merge({
-      id: action.get('api_id'),
-      title: action.get('title'),
-      description: action.get('description'),
-      createdAt: action.get('created_at'),
-      updatedAt: action.get('updated_at')
-    }, ...changes);
-  }
+  describe('/api/actions/:id', function() {
+    testMethodsNotAllowed('/actions/1', allowedMethodsFor['/:id']);
+  });
+
+  describe('GET /api/actions/:id', function() {
+    let action, theme;
+    beforeEach(async function() {
+      action = await actionFixtures.action();
+      await action.load('theme');
+      theme = getExpectedTheme(action.related('theme'));
```

Comment:
The naming of the variable is confusing (`theme` should be `expectedTheme`). But aside from that, I don't think it's a good idea to perform `getExpectedTheme` here (in `beforeEach`):

* You don't need it for 2 out of 3 tests in this block.
* I think it's clearer to use both `getExpectedAction` and `getExpectedTheme` where the expectation is.
* These `getExpected*` functions are written with a `...changes` argument specifically to be customizable for each test. If someday you want to use it for another test in this block with different options, you'll have to remove it from the `beforeEach` anyway.

Reply:
Yep, this is fixed in the following PR.